plugins/main/bridge/bridge.go

```diff
@@ -589,6 +590,30 @@ func cmdDel(args *skel.CmdArgs) error {
 		}
 	}
 
+	// Remove bride if prunening is required
+	if n.Prune == true {
+		br, errprune := bridgeByName(n.BrName)
+
+		if errprune == nil {
+			brIndex := br.Attrs().Index
```

Comment:
Shouldn't this index be read from link gathered in LinkList below? In theory it LinkList could return other index for this bridge (if it was in between deleted/readded)? Very edge situation, so i'm not insisting on that, but imo you should once use LinkList, then double reiterate over it - once to find bridge, second time - using the same list - to step over master indexes of all links.

Reply:
okey. changed.